docs/run-it-mocker-server.md

```diff
@@ -46,10 +46,10 @@ CONFIG=moclojer.yml java -jar moclojer.jar
 ## **Docker**
 
 ```shell
-docker pull ghcr.io/avelino/moclojer:latest
```

Comment:
I'm pretty sure that either avelino and moclojer ghcr owner works fine.

I tried pulling both containers and they worked fine. Any reasons to not use the moclojer ghcr owner?